lib/orderbook/OrderBookRepository.ts

```diff
@@ -1,25 +1,24 @@
 import { Op } from 'sequelize';
 
-import { orders, pairs } from '../types';
+import { db } from '../types';
 import Logger from '../Logger';
-import baseRepository from '../db/baseRepository';
+import DB from '../db/DB';
+import Bluebird from 'bluebird';
 
 type Orders = {
-  buyOrders: orders.dbOrder[];
-  sellOrders: orders.dbOrder[];
+  buyOrders: db.OrderInstance[];
+  sellOrders: db.OrderInstance[];
 };
 
 class OrderbookRepository {
   logger: Logger = Logger.global;
-  models: any;
 
-  constructor(db) {
+  constructor(private db: DB) {
     this.logger = Logger.global;
-    this.models = db.getModels();
```

Comment:
This class  seems to only use `db.models` but never `db` itself. Maybe go back to just having `models` as a class property and drop `db`? We can also change all references to this constructor and have it pass `db.models` instead of `db`. It's also slightly confusing to have `db` as an import as well as `this.db` as a property.

Reply:
I did it because I thought that calling on `models` is confusing, without having the `db.models` context. But I can change it in favor of passing `db.models` to the constructor instead.

Reply:
I did it because I thought that calling on `models` is confusing without having the `db.models` context. Passing `models` to constructor is not the best approach IMO, specially when you need to pass it along to inner layers (xud->orderbook->orderbookRepository). I can change it back to hold `models` as a class property, and keep passing `db` to constructor. we'll still need to import the `DB` type anyway.

Reply:
Cool, that approach sounds good to me.

Reply:
Fixed it, but had to keep `db` as a class member on `P2PRepository` because it has a raw query method (we can wrap it, but it seems to me like an overkill for now).